apis/core/lib/storage/http.ts

```diff
@@ -0,0 +1,30 @@
+import type { MediaObject } from '@rdfine/schema'
+import fetch from 'node-fetch'
+import { Readable } from 'stream'
+import type { MediaStorage } from './types'
+
+export const httpStorage: MediaStorage = {
+  async getStream(media: MediaObject): Promise<Readable> {
+    const url = getUrl(media)
+    const response = await fetch(url)
+
+    return response.body as Readable
+  },
+
+  // Cannot delete remote file
+  async delete() {
+    return null
```

Comment:
Maybe throw if someone tries?

Reply:
No the idea is that I can call `storage.delete()` on any storage. But when you delete an associated media stored in a remote URL, it doesn't do anything for obvious reasons ;)

Reply:
The comment is not very clear. I'll rephrase.